keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -359,14 +365,14 @@ details). For now, we leave it here.
 
 _This section must be completed when targeting beta graduation to a release._
```

Comment:
Again - remove.

Reply:
I have locked the feature gate to true, and removed the unnecessary content @soltysh